eZ/Publish/Core/Limitation/ParentContentTypeLimitationType.php

```diff
@@ -234,4 +235,26 @@ public function valueSchema()
     {
         throw new \eZ\Publish\API\Repository\Exceptions\NotImplementedException(__METHOD__);
     }
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\ValueObject $contentInfo
```

Comment:
this (and type hint) should always be a contentInfo object, in this case the one in API from what I can see from the diff